src/BookingServiceAdmin/BookingAdminGateway.cs

```diff
@@ -19,5 +19,8 @@ public partial class BookingServiceAdminGateway : Gateway, IBookingServiceAdminG
 
         public async Task<HttpResponse<Booking>> GetBooking(Guid bookingId) =>
             await GetAsync<Booking>($"{BookingEndpoint}/{bookingId}");
+
+        public async Task<HttpResponse<bool>> CancelBooking(Note model) =>
```

Comment:
This needs to be a Request type model like the other Post and Patch endpoints in other Controllers